src/cryptonote_basic/hardfork.cpp

```diff
@@ -75,6 +75,7 @@ static constexpr HardFork::Params mainnet_hard_forks[] =
   { network_version_15_lns,                 496969, 0, 1585105200 }, // 2020-03-25 14:00AEDT (03:00UTC)
   { network_version_16_pulse,               641111, 0, 1602464400 }, // 2020-10-12 12:00AEDT (01:00UTC)
   { network_version_17,                     641111 + BLOCKS_EXPECTED_IN_DAYS(180), 0, 1602464400 + 180*24*60*60 }, // HF16 + 180 days
+  { network_version_18,                     641111 + BLOCKS_EXPECTED_IN_DAYS(200), 0, 1602464400 + 200*24*60*60 }, // HF16 + 200 days
 };
```

Comment:
To confirm and finalise actual HF times

Reply:
Also when we do bump this there's no need to keep that BLOCKS_EXPECTED_IN_DAYS stuff for the new HF -- that was just there to be explicit that the fork was deliberately 180 days (for the block reward expiry).